articles/mysql/flexible-server/tutorial-php-database-app.md

```diff
@@ -153,11 +153,26 @@ Pricing for the create resources is as follows:
 - The virtual network doesn't incur a charge unless you configure extra functionality, such as peering. See [Azure Virtual Network pricing](https://azure.microsoft.com/pricing/details/virtual-network/).
 - The private DNS zone incurs a small charge. See [Azure DNS pricing](https://azure.microsoft.com/pricing/details/dns/). 
 
-#### How do I connect to the MySQL database that's secured behind the virtual network with other tools?
 
-- For basic access from a commmand-line tool, you can run `mysql` from the app's SSH terminal.
-- To connect from a desktop tool like MySQL Workbench, your machine must be within the virtual network. For example, it could be an Azure VM that's connected to one of the subnets, or a machine in an on-premises network that has a [site-to-site VPN](../../vpn-gateway/vpn-gateway-about-vpngateways.md) connection with the Azure virtual network.
-- You can also [integrate Azure Cloud Shell](../../cloud-shell/private-vnet.md) with the virtual network.
+### How do I connect to a MySQL database that's secured behind a virtual network?
+
+To connect to a MySQL database, you can use several methods based on the tools and environments at your disposal:
+
+1. **Command-line tool access**: 
+   - Use the `mysql` command from the app's SSH terminal for basic access.
+2. **Desktop tools (for example, MySQL Workbench)**:
+   - **Using SSH tunneling with Azure CLI**:
+     - Create an [SSH session](../../app-service/configure-linux-open-ssh-session.md#open-ssh-session-from-remote-shell) to the web app by using the Azure CLI.
+     - Use the SSH session to tunnel the traffic to MySQL.
+   - **Using site-to-site VPN or Azure VM**:
+     - Your machine must be part of the virtual network.
+     - Consider using:
+       - An Azure VM linked to one of the subnets.
+       - A machine in an on-premises network that has a [site-to-site VPN connection](../../vpn-gateway/vpn-gateway-about-vpngateways.md) to the Azure virtual network.
+4. **Azure Cloud Shell integration**:
```

Comment:
```suggestion
- **Command-line tool access**: 
   - Use the `mysql` command from the app's SSH terminal for basic access.
- **Desktop tools (for example, MySQL Workbench)**:
   - **Using SSH tunneling with Azure CLI**:
     - Create an [SSH session](../../app-service/configure-linux-open-ssh-session.md#open-ssh-session-from-remote-shell) to the web app by using the Azure CLI.
     - Use the SSH session to tunnel the traffic to MySQL.
   - **Using site-to-site VPN or Azure VM**:
     - Your machine must be part of the virtual network.
     - Consider using:
       - An Azure VM linked to one of the subnets.
       - A machine in an on-premises network that has a [site-to-site VPN connection](../../vpn-gateway/vpn-gateway-about-vpngateways.md) to the Azure virtual network.
- **Azure Cloud Shell integration**:
```
Replacing numbers with bullets since the list is not sequential.